src/main/scala/cromwell/backend/BackendValidationActor.scala

```diff
@@ -0,0 +1,50 @@
+package cromwell.backend
+
+import akka.actor.{Actor, ActorLogging}
+import akka.event.LoggingReceive
+import cromwell.backend.BackendValidationActor.{Validate, ValidationResult}
+import wdl4s.{WorkflowCoercedInputs, NamespaceWithWorkflow}
+
+import scala.concurrent.Future
+
+object BackendValidationActor {
+  sealed trait BackendValidationActorMessage
+  case class Validate(namespace: NamespaceWithWorkflow, wfInputs: Option[WorkflowCoercedInputs] = None, wfOptionsJson: Option[String] = None) extends BackendValidationActorMessage
+  case class ValidationResult(isSuccess: Boolean) extends BackendValidationActorMessage
```

Comment:
The logging would be the responsibility of the concrete backends as far as i think.. but in general this looks better with being more verbose. I'll change it. Thanks!